klaxon/src/main/kotlin/com/beust/klaxon/Render.kt

```diff
@@ -17,7 +17,41 @@ object Render {
             is Pair<*, *> -> renderValue(v.second, result.renderString(v.first.toString()).append(": "), prettyPrint, canonical, level)
             is Double, is Float ->
                 result.append(if (canonical) decimalFormat.format(v) else v.toString())
-            else -> result.append(v.toString())
+            is Int, is Boolean -> result.append(v.toString())
```

Comment:
This feels like repetition of some other serialization code somewhere else in Klaxon, but haven't looked in details yet.

Reply:
Right, so why not find the converter for this value and invoke `toJson()` on it?